### PR TITLE
Allow subclassing ToggleButton and MenuButton

### PR DESCRIPTION
**Pull request by nacho**
A couple of patches to allow subclassing Toggle and Menu buttons.

### CODE REVIEW

**Review by GuillaumeGomez — APPROVED** [review]

**Comment by GuillaumeGomez** [comment]
Thanks! Do you see anything missing @sdroege @bilelmoussaoui ?

**Review by sdroege — APPROVED** [review]
looks good to me

**Review by mbiggio — APPROVED** [review]

File: gtk/src/subclass/toggle_button.rs [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+}
+
+unsafe impl<T: ToggleButtonImpl> IsSubclassable<T> for ToggleButton {
+    fn class_init(class: &mut glib::Class<Self>) {
+        Self::parent_class_init::<T>(class);
+
+        if !crate::rt::is_initialized() {
+            panic!("GTK has to be initialized first");
+        }
+
```
Comment (bilelmoussaoui) [inline gtk/src/subclass/toggle_button.rs:43]: If widget has this check already, no need to replicate it on every child widget

**Comment by GuillaumeGomez** [comment]
Thanks!

**Comment by GuillaumeGomez** [comment]
Btw, are there other missing subclassing stuff?

**Comment by bilelmoussaoui** [comment]
> Btw, is there other missing subclassing stuff?

Probably a lot

**Comment by GuillaumeGomez** [comment]
Ok, how do you know you can subclass something then?

**Pull request merged**